app/models/stage.rb

```diff
@@ -28,6 +28,7 @@ class Stage < ActiveRecord::Base
   default_scope { order(:order) }
 
   validates :name, presence: true, uniqueness: { scope: [:project, :deleted_at] }
+  validates :notify_email_address, format: /\A([^;]+@[^;]+)(; ?([^;]+@[^;]+)?)*\z/, allow_blank: true
```

Comment:
wow, is this format documented? I know its normal for emails but nothing else is separated by semicolons. Also, could replace ` ?` by `\W*` if I understand the intention, and maybe replace `[^;]` by `[^\W;]` (or maybe we shouldn't do this by regex?)